packages/swatch/README.md

```diff
@@ -116,6 +118,7 @@ import { Swatch } from '@spectrum-web-components/swatch';
         rounding="full"
         shape="rectangle"
         mixed-value
+        aria-checked="mixed"
```

Comment:
The internal logic should be enough to cover this delivery. Adding it manually here could confuse consumers as to their responsibilities with this pattern. If for some reason these are not getting the attribute at runtime after the below change, then that’s a sign there’s more work to do.

Reply:
Make sense. Thank you